src/app/components/loading-rectangle.tsx

```diff
@@ -1,11 +1,11 @@
-import { Box, BoxProps } from '@stacks/ui';
+import { Box, BoxProps } from 'leather-styles/jsx';
 
 export function LoadingRectangle(props: BoxProps) {
   return (
     <Box
-      backgroundImage="linear-gradient(90deg, rgba(219,219,219,1) 0%, rgba(192,192,247,0.5) 35%, rgba(219,219,219,1) 100%)"
-      backgroundColor="rgb(219,219,219)"
       animation="shine 5s infinite linear"
+      backgroundColor="rgb(219,219,219)"
```

Comment:
Maybe we can ask the designers to give us color codes for this?

Reply:
There might be one in Figma already, will check.